src/python/gudhi/delaunay_complex.pyx

```diff
@@ -176,36 +179,38 @@ cdef class AlphaComplex(DelaunayComplex):
+                            default_filtration_value: bool = False,
+                            square_root_filtrations: bool = False):
         """
-        :param max_alpha_square: The maximum alpha square threshold the simplices shall not exceed. Default is set to
-            infinity, and there is very little point using anything else since it does not save time.
-        :type max_alpha_square: float
-        :param default_filtration_value: [Deprecated] Default value is `False` (which means compute the filtration
-            values). Set this value to `True` if filtration values are not needed to be computed (will be set to
-            `NaN`), but please consider constructing a :class:`~gudhi.DelaunayComplex` instead.
-        :type default_filtration_value: bool
-        :returns: A simplex tree created from the Delaunay Triangulation. The vertex `k` corresponds to the k-th input
+        Args:
+            max_alpha_square: The maximum alpha square threshold the simplices shall not exceed. Default is set to
+                infinity, and there is very little point using anything else since it does not save time.
+            default_filtration_value: [Deprecated] Default value is `False` (which means compute the filtration
+                values). Set this value to `True` if filtration values are not needed to be computed (will be set to
+                `NaN`), but please consider constructing a :class:`~gudhi.DelaunayComplex` instead.
+            square_root_filtrations: Square root filtration values when True. Default is False.
+        Returns:
+            SimplexTree: A simplex tree created from the Delaunay Triangulation. The vertex `k` corresponds to the k-th input
             point. The vertices may not be numbered contiguously as some points may be discarded in the triangulation
             (duplicate points, weighted hidden point, ...).
-        :rtype: SimplexTree
         """
         filtration = 'alpha'
         if default_filtration_value:
             filtration = None
             warnings.warn('''Since Gudhi 3.10, creating an AlphaComplex with default_filtration_value=True is deprecated.
                           Please consider constructing a DelaunayComplex instead.
                           ''', DeprecationWarning)
-        return super().create_simplex_tree(max_alpha_square, filtration)
+        return super().create_simplex_tree(max_alpha_square, filtration, square_root_filtrations)
 
-    def get_point(self, vertex):
+    def get_point(self, vertex: int) -> list[float]:
         """This function returns the point corresponding to a given vertex from the :class:`~gudhi.SimplexTree` (the
         same as the k-th input point, where `k=vertex`)
 
-        :param vertex: The vertex.
-        :type vertex: int
-        :rtype: list of float
-        :returns: the point.
+        Args:
+            vertex: The vertex.
+        Returns:
+            the point.
```

Comment:
> Is it costly to make so that AlphaComplexes do not reorder points (in the simplextree at least) ? so that this function becomes non-necessary ?

That's already the case, the doc you are quoting says "same as the k-th input point". This function is a left-over from an old release where points could be shuffled (only if they were in dimension 3 IIRC). We could hide this function in the doc if it causes too much confusion.

> There is also the question of removed points, but 
>     1. is it generically not the case ?

With weights, it is perfectly generic for points to disappear.

>     2. we can also "recreate" the deleted point in the simplextree (with a flag enabled by default)

I don't understand what you mean by that. If we just add an extra vertex connected to nothing, that will add a connected component, that's much more problematic than a "missing" point.

> For the return type I think it is better to return a numpy array or "à défaut" a tuple.

For such a legacy function that mostly remains to avoid breaking old user code, I am not convinced it is worth changing it and risking to break said user code.

> In general, in python I feel that the classes `RipsComplex`, `AlphaComplex`, etc. are not that useful in the interface, and could be replaced by helper functions, directly returning the simplextree; but that's another topic.

I think @VincentRouvreau must be tired of me saying exactly that for years now :wink: The class is not just unnecessary for users, it also complicates the implementation because half of the work is done in the constructor, forcing us to store intermediate stuff for later use in `create_simplex_tree`.